runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -622,6 +626,30 @@ else if (killType == KillType.COX_CM)
 		takeScreenshot(fileName, screenshotSubDir);
 	}
 
+	@Subscribe
+	public void onScriptPostFired(ScriptPostFired scriptPostFired)
+	{
+		if (scriptPostFired.getScriptId() != ScriptID.NOTIFICATION_START)
+		{
+			return;
+		}
+		int[] is = client.getIntStack();
+		int scriptId = is[0];
+		int arg = is[1];
+		if (scriptId != ScriptID.NOTIFICATION_DELAY || arg != 0) // !notification_delay(0)
```

Comment:
If the argument's value is to be checked here, it should probably be documented in `ScriptID.NOTIFICATION_DELAY`'s javadoc.